src/main.js

```diff
@@ -2,6 +2,8 @@
 
 const INPUT_BUTTON_TYPES = ['button', 'submit', 'reset'];
 
+document.getElementById('comments').defaultValue = '';
```

Comment:
Ohhh, you decided to clear the text area by js code. Good practice👍

Reply:
I`ve been googling it so long